src/scss/_tokens.scss

```diff
@@ -34,8 +34,18 @@ $spacers: (
   1: 10px,
   2: 20px,
   3: 40px,
-  4: 60px,
-  5: 96px,
+  // 4: 60px,
+  // 5: 96px,
+  // absolute sizes
+  8: 8px,
+  12: 12px,
+  16: 16px,
+  20: 20px,
+  28: 28px,
+  40: 40px,
```

Comment:
what's the thinking for having different tokens for the same value, e.g. `40`/`3` for `40px`?

Reply:
got it, I'm seeing the `-40` utilities in the deployment so I'll approve!